prow/cmd/peribolos/main.go

```diff
@@ -655,7 +701,7 @@ type editTeamClient interface {
 }
 
 // configureTeam patches the team name/description/privacy when values differ
-func configureTeam(client editTeamClient, orgName, teamName string, team org.Team, gt github.Team) error {
+func configureTeam(client editTeamClient, orgName, teamName string, team org.Team, gt github.Team, parent *int) error {
```

Comment:
I do not understand the motivation behind / value of changing this to `parent *int` instead of `int`.

Reply:
`0` is technically a valid parent team ID. We need to be able to tell the difference between `0` and `nil`.

Reply:
got it, thanks! Can you include a unit test for removing the parent from a team then?